stdlib/3.4/enum.pyi

```diff
@@ -18,7 +19,7 @@ class EnumMeta(ABCMeta, Iterable[Enum], Sized, Reversible[Enum], Container[Enum]
     def __members__(self: Type[_T]) -> Mapping[str, _T]: ...
 
 class Enum(metaclass=EnumMeta):
-    def __new__(cls: Type[_T], value: Any) -> _T: ...
+    def __new__(cls: Type[_T], value: str, names: Optional[str], module: Optional[str], qualname: Optional[str], type: Optional[Type[_T2]], start: Optional[int]) -> _T: ...
```

Comment:
I don't think this is right—this method is for constructing enum instances, not enum classes. If anything, your change should be on EnumMeta.

Reply:
So maybe we can't actually do much to fix this in typeshed.

Reply:
I must admit I find the `Enum` class rather baffling, I will look more into it